eduNEXT/eox-tenant
# perf(DS-365): olive support

These commits:
- [PERF] Remove the useless backends for versions older than maple.
- [FEAT] Add compatibility for Python, 3.10 y 3.11.
- [FEAT] Add compatibility for Django 32.
- [PERF] Remove compatibility for django 22.
- [CI] Add the new python and django versions to the matrix test in the github actions workflow.

Nota:
The python and django versions were upgraded following this document: https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit#gid=195838733

### How to test
You should have an tutor environment and install eox-tenant as a private package.
Then you should run the test cases defined in this document: https://docs.google.com/document/d/1921hOajxAYrdin_thEZ005aQBwPNoj2zsbgkf3q1Mjc/edit.

### Info
How to add eox-tenant: https://docs.tutor.overhang.io/dev.html#xblock-and-edx-platform-plugin-development
Create user for tutor: https://docs.tutor.overhang.io/local.html#creating-a-new-user-with-staff-and-admin-rights
How to configure multitenant: https://docs.google.com/document/d/1921hOajxAYrdin_thEZ005aQBwPNoj2zsbgkf3q1Mjc/edit#heading=h.doulrbfn1rer
